torch/distributed/tensor/_ops/_tensor_ops.py

```diff
@@ -804,3 +804,70 @@ def size_split(N, i) -> list:
         for _ in range(len(output_size_list))
     ]
     return OutputSharding(output_spec_list)
+
+
+@register_op_strategy(aten.split.Tensor, schema_info=RuntimeSchemaInfo(1))
+def split_strategy(op_schema: OpSchema) -> TupleStrategy:
+    input_strategy = op_schema.args_schema[0]
+    assert isinstance(input_strategy, OpStrategy)
+
+    split_size_or_sections = op_schema.args_schema[1]
+
+    dim = op_schema.args_schema[2] if len(op_schema.args_schema) > 2 else 0
+    assert isinstance(dim, int)
+    dim = normalize_dim(dim, input_strategy.ndim)
+
+    def size_split(N, i) -> list:
```

Comment:
Can you be more specific about the typing here for size_split? at least preserve the input type with a type var.